README.md

```diff
@@ -33,4 +37,32 @@ TBD
 
 ## Documentation
 
-Architectural Decision Records are stored in `doc/adr`
+Architectural Decision Records (ADR) are stored in `doc/adr`
+To create a new ADR, first install [ADR-tools](https://github.com/npryce/adr-tools) if you don't
+already have it installed.
+* `brew install adr`
```

Comment:
Their documentation calls the package `adr-tools` : `brew install adr-tools`

https://github.com/npryce/adr-tools/blob/master/INSTALL.md

Reply:
oopsies thanks!